build-tools/automation/templates/core-tests.yaml

```diff
@@ -1,7 +1,7 @@
 parameters:
   condition: succeeded()
   runNativeTests: false
-  runJavaTests: false
+  runJavaTests: true
```

Comment:
With `java-source-utils` tests now passing on Windows, I think we can remove this parameter *entirely*, along with all usage of it, as every callsite should now be `true`.

Reply:
Removed.